pb_app/lib/submission_form.dart

```diff
@@ -84,8 +86,12 @@ class _SubmissionFormScreenState extends State<SubmissionFormScreen> {
       ),
       body: PageView(
         children: const [
-          _Card('Home Screen'),
-          _Card('Lock Screen'),
+          _Card(
+            'Home Screen',
+          ),
+          _Card(
+            'Lock Screen',
+          ),
```

Comment:
```suggestion
          _Card('Home Screen'),
          _Card('Lock Screen'),
```

Best to leave out as many style changes as possible :)